compiler/optimizer/Dominators.cpp

```diff
@@ -110,15 +124,22 @@ TR_Dominators::TR_Dominators(TR::Compilation *c, bool post) :
             traceMsg(comp(), "Some blocks are not reachable from exit. Post-dominator info is invalid.\n");
          return;
          }
+      if (!acceptUnreachableBlocks)
+         {
+         _hasUnreachableBlocks = true;
+         if (trace())
+            traceMsg(comp(), "Unreachable block in the CFG %d %d\n", _topDfNum, _numNodes-1);
```

Comment:
Can this message mention that the dominators are therefore invalid, like in the postdominator case?